### PR TITLE
Set up Project - Backend

### LINKED ISSUE #1: [2pt] Set up the project for both the backend and frontend [Group Task]
- Set up the project for both the backend and frontend

### PR DESCRIPTION
**Pull request by rubydevi**
## Description
We have set up the rails app and added the linters

## Changes made 

- Created rails API app using `rails new . --api -d postgresql` command
- We have also set up `rubocop`
- Updated Kanban Progress
  image
## PR Link for Frontend
- https://github.com/rubydevi/jetlogix-frontend/pull/1

### CODE REVIEW

**Comment by Yonashenok** [comment]
i am reviewing this PR Guys

**Review by Yonashenok — APPROVED** [review]
## Hi @RileyManda  and @rubydevi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [x] Correct Git-flow followed :100: 
- [x] the project setup correctly 💯 
- [x] README file is professional :rocket: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+
+### Install
+
+Install this project with:
+
+    ```sh
+    cd my-project
+    bundle install
+    ```sh
```
Comment (Yonashenok) [inline README.md]: please kindly fix the installation section. every thing look cool nice work
image

Comment (rubydevi) [reply README.md]: Thanks 👍 I'll definitely do that.

**Pull request merged**